src/formElementPlugins/fileUpload/constants.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  ERROR_FILE_ALREADY_UPLOADED: 'File already uploaded',
```

Comment:
hmm not a required change but I think this would need to be parameters of the plugin (localised when configuring plugin)

Reply:
Do you mean for all strings associated with a plugin? I did it like this also for Checkbox, etc. Thought this would do until we decide an "official" localisation strategy for plugins. (FWIW, I was imagining each plugin would have their own localisation files, but they'd "inherit" the language setting from the main app, however we do that.)

Reply:
yeah let's flag it as discussion: https://github.com/openmsupply/application-manager-web-app/issues/455